src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -42,6 +42,7 @@ public function getConfigTreeBuilder()
                 ->scalarNode('version')->defaultValue('0.0.0')->info('The version of the API.')->end()
                 ->scalarNode('default_operation_path_resolver')->defaultValue('api_platform.operation_path_resolver.underscore')->info('Specify the default operation path resolver to use for generating resources operations path.')->end()
                 ->scalarNode('name_converter')->defaultNull()->info('Specify a name converter to use.')->end()
+                ->scalarNode('eager_loading_max_joins')->defaultValue(100)->info('Max number of joined relations before EagerLoading throws a RuntimeException')->end()
```

Comment:
From what I can see from the implementation above, `$depth` is the *level* of joined relations, not the *number* of joined relations. So this is misleading, and I suspect 100 is too liberal.